cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -690,7 +690,13 @@ def test_cluster_section_from_file(mocker, config_parser_dict, expected_dict_par
         ("extra_json", None, {}, None),
         ("extra_json", "", {}, None),
         ("extra_json", "{}", {}, None),
-        ("extra_json", "{'test': 'test'}", {"test": "test"}, None),
+        ("extra_json", '{"test": "test"}', {"test": "test"}, None),
+        (
+            "extra_json",
+            "{'test': 'test'}",
+            {"test": "test"},
+            None,
+        ),  # WARNING it is considered a valid value by yaml.safe_load
         ("extra_json", "{'test': 'test'", None, "Error parsing JSON parameter"),
         ("extra_json", "fake_value", "fake_value", None),  # WARNING it is considered a valid value by yaml.safe_load
```

Comment:
I know it is not related to your patch but could you please also remove this WARNING since an empty string is a valid JSON param. Thanks

Reply:
OK